handleData.js

```diff
@@ -119,11 +127,52 @@ var Character = function(realName, gameName, level, gameClass, race) {
     this.characterLevel = level;
     this.characterClass = gameClass;
     this.characterRace = race;
+    this.inventory = [];
+    this.money = { "Copper Pieces"      : 0,
+                   "Silver Pieces"      : 0,
+                   "Gold Pieces"        : 0,
+                   "Platinum Pieces"    : 0,
+                   "Electrum Pieces"    : 0};
```

Comment:
Okay, as long as there isn't a syntax issue with a trailing comma, then I don't see the harm.

I don't yet see the advantage, probably because I haven't tried sorting the contents of any objects here. I'm a little confused about that, at the moment, but it may be tiredness setting in.

And okay, no enum for currencies!